### DISCUSSION TITLE
Osu lazer crashed randomly

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17316] [Q&A] [answered]
Osu lazer just crashed, game was open since this morning but everything seemed fine. So I played a couple maps, and after a while I pressed start on a map and poof, game gone without any notice.

Hopefully these are osu lazer log files, hopefully they contain the answer somewhere
Odd
[performance.log](https://github.com/ppy/osu/files/8301931/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8301932/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8301933/updater.log)
[database.log](https://github.com/ppy/osu/files/8301934/database.log)
[network.log](https://github.com/ppy/osu/files/8301935/network.log)

**Comment by peppy** [comment, marked as answer]
If the game disappears with no warning, it is 100% going to be a GPU driver level issue. In these cases you can check the log files and notice that they don't contain an error. You will need to check event viewer to find the actual cause.

**Reply by AuracleTech** [reply]
![image](https://user-images.githubusercontent.com/40531871/158947941-33401fde-9d58-41ea-a4bb-eb445570aa66.png)
It appears to be a .NET error
![image](https://user-images.githubusercontent.com/40531871/158947970-c059d4ce-e958-4acb-91a8-d2ee4b2d8065.png)

**Reply by bdach** [reply]
that's a generic access violation error and is likely to trace back to drivers. please update your GPU drivers and try again.

**Reply by AuracleTech** [reply]
![image](https://user-images.githubusercontent.com/40531871/158949246-6a795b7a-10dc-44ac-907e-611daf67dda2.png)
Already up to date. Well... I'll just re-open if it happens...
Im sad I have so many issues with lazer, and only lazer, no other game

**Reply by peppy** [reply]
> Im sad I have so many issues with lazer, and only lazer, no other game

The usual reasoning for this is that it is OpenGL while most other games are not. And drivers have very shoddy opengl support at times.

That said, your failure looks like something else potentially not driver related. Hard to say what.
